region/brinstar/green/Early Supers Room.json

```diff
@@ -496,6 +496,31 @@
         "Note that if Samus takes more than 35 seconds after entering the room, before shooting the ceiling, the Waver will not come down."
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Morph",
+        {"refill": ["Energy"]},
```

Comment:
Might have to `OR:[canBePatient, h_disableEtanks]` in the case of big Etank numbers.  Even then I don't know how long that takes.